examples/type2_type3.py

```diff
@@ -5,7 +5,7 @@
 import os
 import matplotlib.pyplot as plt  # Grab MATLAB plotting functions
 from control.matlab import *     # MATLAB-like functions
-from scipy import pi
+from numpy import pi
```

Comment:
This has already been resolved in #965. Please rebase.

Reply:
Ok, I tried again.

Reply:
Nope.

1. Fast forward your main branch to upstream/main
2. In system-norms use `git rebase -i main`

https://github.com/python-control/python-control/wiki/How-to-contribute-with-a-pull-request

This gets rid of the extra commits which are already in the main branch. While you are at it, you can also squash some of the commits and use better messages than "Update sysnorm.py"

Reply:
I did steps 7-9:
>... `git checkout main; git fetch --all; git merge origin/main
git push
and then bring those changes into your branch:

git checkout <my-new-branch-name>; git rebase main`

Is it step 12 I should do?
`git checkout main; git fetch --all; git merge upstream/main
`

Reply:
I am afraid your multiple merges and manual fixes like 7a5af505c4996056b033a915e0171ea9d132a5dc have created a situation which git cannot resolve automatically:

![image](https://github.com/python-control/python-control/assets/4623504/53b1137c-e1de-4a7f-80bf-137e2a87a8b8)


```
[ben@skylab:~/src/python-control]% git status                                                             [0]
On branch system-norms
Your branch is up to date with 'henriks76/system-norms'.

nothing to commit, working tree clean
[ben@skylab:~/src/python-control]% git rebase main                                                        [0]
Auto-merging control/sysnorm.py
CONFLICT (add/add): Merge conflict in control/sysnorm.py
error: could not apply fd076c5... New function for LTI system norm computation
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply fd076c5... New function for LTI system norm computation
[ben@skylab:~/src/python-control]% git status                                                             [1]
interactive rebase in progress; onto 85328d9
Last commands done (10 commands done):
   pick 49f7e5f Update sysnorm.py
   pick fd076c5 New function for LTI system norm computation
  (see more in file .git/rebase-merge/done)
Next commands to do (9 remaining commands):
   pick 0fe2c57 * Updated documentation of function norm * Added control/tests/sysnorm_test.py
   pick cdf4bab Update sysnorm.py
  (use "git rebase --edit-todo" to view and edit)
You are currently rebasing branch 'system-norms' on '85328d9'.
  (fix conflicts and then run "git rebase --continue")
  (use "git rebase --skip" to skip this patch)
  (use "git rebase --abort" to check out the original branch)

Unmerged paths:
  (use "git restore --staged <file>..." to unstage)
  (use "git add <file>..." to mark resolution)
        both added:      control/sysnorm.py

no changes added to commit (use "git add" and/or "git commit -a")
```

You have to go through it step by step, because only you know what is relevant.